### DISCUSSION TITLE
Quick documentation (Ctrl-Q) failed with new UI using ideavim

### DISCUSSION THREAD

**Discussion by Marchonq** [JetBrains/ideavim#652] [Q&A]
I mapped the action in ideavimrc file like:
nmap <leader>q <Action>(QuickJavaDoc)
no pop-up window is shown in new UI, but is ok in old UI.
I'm using Rider 2022.3, could this problem be fiexed on ideavim side? Or is this problem is later Rider versions?

**Comment by ljchen1** [comment]
Hi,

I saw that you have already asked this in a JetBrains ticket and I advised you to update to the latest Rider/IdeaVIM because it's working fine there from my testing.
Please let me know if it still doesn't help.
